libs/galley-types/src/Galley/Types/Teams.hs

```diff
@@ -199,6 +208,13 @@ data TeamUpdateData = TeamUpdateData
     , _iconKeyUpdate :: Maybe (Range 1 256 Text)
     } deriving (Eq, Show)
 
+data TeamSettings = TeamSettings
+    { _tsUserTokenTimeout     :: ZAuth.UserTokenTimeout
```

Comment:
I just used the existing name; but I'm starting to think that `UserTokenExpiry` may be more descriptive ya?